haddock.cabal

```diff
@@ -64,7 +64,7 @@ executable haddock
 
   -- haddock typically only supports a single GHC major version
   build-depends:
-    base ^>= 4.14.0.0
+    base ^>= 4.12.0.0
```

Comment:
Wouldn't this also need to admit 4.13 since this is what we use when we build with stage1?

Reply:
ah good catch, the peril of just trying with what `ghc.nix` gives you!